pages/linux/sport.md

```diff
@@ -0,0 +1,32 @@
+# sport
+
+> Search and install SlackBuilds.
+> More information: <http://slackermedia.info/handbook/doku.php?id=slackbuilds>.
+
+- Pull the list of SlackBuilds to run `sport` for the first time:
+
+`sudo mkdir -p /usr/ports && sudo rsync -av rsync://slackbuilds.org /slackbuilds/$(awk '{print $2}' /etc/slackware-version)/ /usr/ports/`
```

Comment:
There's so much complexity in this command. Is this really useful? If so, that's ok.

Reply:
In Slackware there are some things that need to be done manually. The handbook in this tldr page said that this command is needed to run `sport` for the first time, so I think it would help the users, specially newcomers to Slackware.

Reply:
Great.